automations/python/label_pr.py

```diff
@@ -197,7 +197,9 @@ def main():
     pr = get_pull_request(gh, args.pr_url)
     log.info(f"Found PR: {pr.title}")
 
-    if pr.labels:
+    # Skip a PR if it already has labels, as long as it has labels that are NOT
+    # a stack label - if all of its labels are stack labels, apply the new labels too
```

Comment:
Perhaps an additional mention here about the _why_—i.e., stack labels are automatically applied—would be useful context.